### ISSUE TITLE
[Bug] yarn install (in pom.xml) timeout in arm64

### ISSUE THREAD

**Issue by aroundabout** [apache/incubator-hugegraph-toolchain#520]
### Bug Type (问题类型)

exception / error (异常报错)

### Before submit

- [X] I had searched in the [issues](https://github.com/apache/hugegraph-toolchain/issues) and found no similar issues.

### Environment (环境信息)

- Toolchain Version: latest
- platform: github action (linux/arm64)

### Expected & Actual behavior (期望与实际表现)

https://github.com/hugegraph/actions/actions/runs/6194003062/job/16817665758#step:5:11833
In repo [hugegraph/actions](https://github.com/hugegraph/actions), build docker images and push:
![image](https://github.com/apache/incubator-hugegraph-toolchain/assets/49650772/dac77992-78c5-459e-aa80-8fa9c10943fc)
The reason is that `yarn install` will timeout in arm64 platform
![image](https://github.com/apache/incubator-hugegraph-toolchain/assets/49650772/8ca1bf0b-5ecd-47a7-88e7-8a9f1be8571f)

The problem is still not solved until 2023.7 as the issue https://github.com/nodejs/docker-node/issues/1335#issuecomment-1652973487

The way to solve the problem is to set the timeout like https://github.com/pomerium/pomerium/pull/3019/files

In toolchain, we can change the line: 
https://github.com/apache/incubator-hugegraph-toolchain/blob/master/hugegraph-hubble/hubble-dist/pom.xml#L84 
to 
```
timeout 120000 yarn install &amp;&amp; yarn build || exit 1
```

**Comment by aroundabout** [comment]
@simon824

**Comment by simon824** [comment]
fixed in #519
